Utilities/StaticAnalyzers/run_static_checks.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#export WORKSPACE=/build/razumov/shift
+#export CMSSW_IB=CMSSW_14_2_X_2024-11-05-2300
+#export NCPU2=4
+#pushd $WORKSPACE/$CMSSW_IB
+#git cms-addpkg --ssh Utilities/StaticAnalyzers
+rm -rf $WORKSPACE/llvm-analysis; mkdir $WORKSPACE/llvm-analysis
```

Comment:
If this script is about testing the static analyzer with the code in `Utilities/StaticAnalyzers/test`, I'd suggest to move the scrip to the `test` directory, rename it to highlight it runs tests (e.g. `run_static_checker_tests.sh`).

I'd also suggest to check all of these environment variables are defined at the beginning of the script (and abort if they are not defined), especially before any "dangerous" commands like `rm -rf`.

Reply:
Thanks, will do.